code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -1044,10 +1042,10 @@ var/global/list/obj/item/device/pda/PDAs = list()
 				if((mode<=9) || (locate(mode) in pda_app_menus))
 					mode = 0
 				else
-					mode = round(mode/10)
-					if(mode==4)//Fix for cartridges. Redirects to hub.
+					mode = round(mode/10)//TODO: fix this shit up
+					if((mode==4) || (mode==5))//Fix for cartridges. Redirects to hub.
 						mode = 0
-					else if(mode >= 40 && mode <= 49)//Fix for cartridges. Redirects to refresh the menu.
+					else if(mode >= 40 && mode <= 53)//Fix for cartridges. Redirects to refresh the menu.

```

Comment:
I've been pulling my hair over how horrendous cartridges app were designed. All the modes should be defines ideally. But fixing all the shit that's wrong with cartridges is beyond the scope of this PR.